src/app/signup/signup.component.html

```diff
@@ -0,0 +1,27 @@
+<div class="split left">
+  <div class="container">
+      <h1 style="font-size: 1.6rem;"><b>First, enter your email</b></h1>
+      <p style="font-size: 1rem;">Just one more email — a quick confirmation — before you say goodbye to overstuffed inboxes for good.</p>
+    
+      <form>
+        <p><strong>Your email address</strong></p>
+        <input type="email" name="email" id="email" placeholder="you@example.com">
+      </form>
+      <br/>
+      <br/>
+      <button mat-raised-button color="primary" class="signup-btn"><b>Continue</b> <i class="material-icons">arrow_forward</i></button>
+  </div>
+</div>
+
+<div class="split right">
```

Comment:
Make the page responsive by using a media query on this div by either hiding it or showing it above the other input div.